honeybee_radiance/primitive/material/trans.py

```diff
@@ -0,0 +1,371 @@
+        """
+        return self._r_reflectance
+
+    @r_reflectance.setter
+    def r_reflectance(self, reflectance):
+        self._r_reflectance = typing.float_in_range(reflectance, 0, 1)
+
+    @property
+    def g_reflectance(self):
+        """Transmissivity for green channel.
+
+        The value should be between 0 and 1 (Default: 0).
+        """
+        return self._g_reflectance
+
+    @g_reflectance.setter
+    def g_reflectance(self, reflectance):
+        self._g_reflectance = typing.float_in_range(reflectance, 0, 1)
+
+    @property
+    def b_reflectance(self):
+        """Transmissivity for blue channel.
+
+        The value should be between 0 and 1 (Default: 0).
+        """
+        return self._b_reflectance
+
+    @b_reflectance.setter
+    def b_reflectance(self, reflectance):
+        self._b_reflectance = typing.float_in_range(reflectance, 0, 1)
+
+    @property
+    def specularity(self):
+        """Fraction of specularity.
+
+        In most cases specularity fractions greater than 0.1 are not realistic
+        (Default: 0).
+        """
+        return self._specularity
+
+    @specularity.setter
+    def specularity(self, spec_value):
+        self._specularity = typing.float_in_range(spec_value, 0, 1)
+
+    @property
+    def roughness(self):
+        """Roughness is specified as the rms slope of surface facets.
+
+        A value of 0 corresponds to a perfectly smooth surface, and a value of 1
+        would be a very rough surface. Roughness values greater than 0.2 are not
+        very realistic. (Default: 0).
+        """
+        return self._roughness
+
+    @roughness.setter
+    def roughness(self, roughness_value):
+        self._roughness = typing.float_in_range(roughness_value, 0, 1)
+
+    @property
+    def transmitted_diff(self):
+        """Transmitted diffuse.
+
+        The transmitted diffuse component is the fraction of transmitted light that is
+        transmitted diffusely in as scattering fashion.
+        """
+        return self._transmitted_diff
+
+    @transmitted_diff.setter
+    def transmitted_diff(self, value):
+        self._transmitted_diff = typing.float_in_range(value, 0, 1)
+
+    @property
+    def transmitted_spec(self):
+        """Transmitted specular.
+
+        The transmitted specular component is the fraction of transmitted light that is
+        not diffusely scattered.
+        """
+        return self._transmitted_spec
+
+    @transmitted_spec.setter
+    def transmitted_spec(self, value):
+        self._transmitted_spec = typing.float_in_range(value, 0, 1)
+
+    @classmethod
+    def from_reflected_specularity(
+            cls, name, r_reflectance=0.0, g_reflectance=0.0, b_reflectance=0.0,
+            reflected_specularity=0.0, roughness=0.0, transmitted_diff=0.0,
+            transmitted_spec=0.0, modifier="void", dependencies=None):
+        """Create trans material from reflected specularity.
+
+        See:
```

Comment:
Is this type of docstring compatible with Sphinx?  Or should it be `Note`?

Reply:
Note is good. I changed it to Note.